packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -1656,7 +1659,7 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
    * Label for determining since which version the changes for timeline menu dropdown are showing.
    * Receives the time label as a parameter (`timestamp`).
    */
-  'timeline.since': 'Since: {{timestamp, datetime}}',
+  'timeline.since': '{{timestamp, datetime}}',
```

Comment:
NIT: Is there any need for this now? I don't imagine the formatting here will be localized

Reply:
Good catch, actually I'm removing that in the changes I'm introducing for this in `corel` I think we can clear it there.